providers/edge/src/airflow/providers/edge/cli/edge_command.py

```diff
@@ -295,7 +295,18 @@ def start(self):
 
             logger.info("Quitting worker, signal being offline.")
             try:
-                worker_set_state(self.hostname, EdgeWorkerState.OFFLINE, 0, self.queues, self._get_sysinfo())
+                if _EdgeWorkerCli.maintenance_mode:
+                    worker_set_state(
+                        self.hostname,
+                        EdgeWorkerState.OFFLINE_MAINTENANCE,
+                        0,
+                        self.queues,
+                        self._get_sysinfo(),
+                    )
+                else:
+                    worker_set_state(
+                        self.hostname, EdgeWorkerState.OFFLINE, 0, self.queues, self._get_sysinfo()
+                    )
```

Comment:
I think this can be made a bit simpler/shorter:
```suggestion
                worker_set_state(
                    self.hostname,
                    EdgeWorkerState.OFFLINE_MAINTENANCE if _EdgeWorkerCli.maintenance_mode else EdgeWorkerState.OFFLINE,
                    0,
                    self.queues,
                    self._get_sysinfo(),
                )
```